composer.json

```diff
@@ -17,6 +17,7 @@
     "yiisoft/yii2": "2.0.*",
     "yiisoft/yii2-twig": "^2.0.4",
     "2amigos/yii2-ckeditor-widget": "^2.0.2",
+    "2amigos/yii2-translateable-behavior": "^1.0.4",
```

Comment:
this should be 1.0.5 when it is released. I noticed that the requirement was missing entirely here.